src/react.tsx

```diff
@@ -1,21 +1,37 @@
 import { DocumentNode } from "graphql"
+import { renderToStaticMarkup } from "react-dom/server"
```

Comment:
better pass this function in to getDataFromTree, as done with the other functions, this makes sure that the package doesn't depend accidentally on react